README-make.md

```diff
@@ -124,10 +124,24 @@ Run `make zope-start` to start the Zope instance.
 
 ### Run tests
 
-This is not in the Makefile yet.  We need to figure out how best to run the tests first.
-A few sample commands that work for me:
+Use the `make test` target to run unit tests for a package:
 
+```bash
+# Run all unit tests for a package (package is required)
+make test package=plone.namedfile
+
+# Run a specific test
+make test package=plone.namedfile test=test_scaling
```

Comment:
Use generic shell syntax highlighter, don't use comments for instructions because they are low contrast in the rendered code block, and make it easier to copy-paste a single command.

````suggestion
Use the `make test` target to run unit tests for a package.

For example, run all unit tests for a given package.

```shell
make test package=plone.namedfile
```

Run a specific test in the given package.

```shell
make test package=plone.namedfile test=test_scaling
````

Reply:
Line 135 describes how to run only one test from a given package.

What I'm trying to say is that it's not possible to execute the command:

`make test`

it's necessary to pass the package:

`make test package=plone.namedfile`

See: https://github.com/plone/buildout.coredev/pull/1059/files#diff-35b89ef65700ec4da068cc82250d658677a5aaabec8b91e21d0d3e69987c949fR130

Reply:
I still don't understand what you mean. Line 135 mentions "the given package", and is followed by line 138, which has the `package` parameter.

Reply:
Line 135 says "the given package," but it doesn't make it clear that this package is required in `make test`. Someone reading this might think it's just an example of running a specific test on a package. And they might try running `make test` thinking it will run all the tests from all the packages. But that won't work.

See: https://github.com/plone/buildout.coredev/pull/1059/files#diff-b2d64a693b39f5304b8f4cfed833e1e499f264e78c5735da98c2d3eb79112777R61

See @davisagli 's https://github.com/plone/buildout.coredev/pull/1059#pullrequestreview-3520219716. It might clarify what I'm trying to say.

Reply:
@stevepiercy as @davisagli  said here https://github.com/plone/buildout.coredev/pull/1059#issuecomment-3591002304, we already have a script that runs all the tests. So I think we can leave the `package` parameter required for now.

Reply:
I checked and actually the target in branch 6.2 doesn't support running only the tests from a specific package.